genai/client.go

```diff
@@ -87,7 +87,15 @@ Import the option package as "google.golang.org/api/option".`)
 	if err != nil {
 		return nil, fmt.Errorf("creating discovery client: %w", err)
 	}
-	gc.SetGoogleClientInfo("gccl", "v"+internal.Version, "genai-go", internal.Version)
+
+	kvs := []string{"gccl", "v" + internal.Version, "genai-go", internal.Version}
```

Comment:
Would it make more sense to call this `WithClientInfo` or something like that, since it's just a key-value pair appended to `SetGoogleClientInfo`? There's nothing about "attribution" anywhere except how the user may interpret it

Reply:
Done.